src/view/configuration/edgeConfigurationsSection.jsx

```diff
@@ -703,4 +703,10 @@ EdgeConfigurationsSection.propTypes = {
   context: PropTypes.object.isRequired,
 };
 
+// Export a function to reset cached values (useful for testing)
+export const resetEdgeConfigurationsCache = () => {
```

Comment:
This was interesting. I don't like it but without it the tests from sandboxScenarios.spec.jsx were not passing. It seems the firstPageOfSandboxes was not resetted between tests in the same file. The sandbox management API call was made just once. If you have other ideas on how to fix this, I am all ears.